src/radio.cpp

```diff
@@ -0,0 +1,45 @@
+#include "radio.hpp"
+
+namespace bzzz
+{   
+    int data[16];
+
+    void readPiData(void)
+    {
+        String dataFromClient;
+        
+        if (Serial.available() > 0) 
+        {
+            dataFromClient = Serial.readStringUntil('\n');
+
+            for (int i = 0; i < 16; i++) 
+            {
+                // take the substring from the start to the first occurence of a comma, convert it to int and save it in the array
+                data[i] = dataFromClient.substring(1, dataFromClient.indexOf(",")).toInt();
+
+                //cut the data string after the first occurence of a comma
+                dataFromClient = dataFromClient.substring(dataFromClient.indexOf(",") + 1);
+            }
+        }
+    }
+
+    void readRadioData(int &radioThrottle, int &radioRoll, int &radioPitch, int &radioYaw, int &radioSwitchC,
+    int &radioVRA, int &radioVRC, int &radioVRB, int &radioArm, int &radioKill, int &radioSwitchD, int &radioVRE) 
+    {
+        readPiData();
```

Comment:
If `readPiData` is invoked every time we call `readRadioData`, then `readPiData` should probably be declared as `static` (i.e., private) so that we cannot invoke it unnecessarily.

Reply:
On second thought, maybe it would be better to remove `readPiData` from here